gravitee-node-container/src/main/java/io/gravitee/node/container/spring/env/AbstractGraviteePropertySource.java

```diff
@@ -86,6 +87,7 @@ private void watchProperty(WatchablePropertyResolver<?> propertyResolver, String
             .defer(() -> propertyResolver.watch(value.toString()))
             .subscribeOn(Schedulers.io())
             .repeat()
+            .delay(1, TimeUnit.SECONDS)
```

Comment:
Can you create a constant (ex: REPEAT_DELAY_SECONDS) or something?
Also, are you able to provide a test that validates the behavior 🙏 ?

Reply:
I created a constant and I added a test... didn't have idea how to test more accurately